### PR TITLE
Zadanie 4 - Map class

### PR DESCRIPTION
**Pull request by mariuszlisowski**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -19,6 +19,7 @@ js/reveal.min.js
 build/
 .vscode/
 googletest/
+compile_flags.txt
```
Comment (mariuszlisowski) [inline .gitignore:22]: I use the clang compiler and need to ignore the compile_flags file.

File: shm/inc/Map.hpp [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "Island.hpp"
+#include <vector>
+
+class Map {
+public:
+
+private:
+    std::vector<Island&> vectorOfIslands_;
```
Comment (mariuszlisowski) [inline shm/inc/Map.hpp:10]: Reference error in type:
In template: 'pointer' declared as a pointer to a reference of type 'Island &'clang(illegal_decl_pointer_to_reference)

File: shm/inc/Map.hpp [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "Island.hpp"
+#include <vector>
```
Comment (mariuszlisowski) [inline shm/inc/Map.hpp:4]: Newline. Compiler directive (pragma) s, user header files ("") and standard headers (<>) should be separated from each other. Any other opinions? Why?

File: shm/inc/Map.hpp [hunk 4]
```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "Island.hpp"
```
Comment (mariuszlisowski) [inline shm/inc/Map.hpp:3]: Newline. Compiler directive (pragma) s, user header files ("") and standard headers (<>) should be separated from each other. Any other opinions? Why?

**Comment by MoreThanADC** [comment]
That's a good code, nothing to add :)

**Review by lukasz-kukulka — APPROVED** [review]

**Pull request merged**